scripts/compare_license.py

```diff
@@ -0,0 +1,167 @@
+"""Script for copyright/license reports
```

Comment:
Maybe this script needs some licensing information for itself.

Reply:
:rofl: Nah it's alright. By default it gets licensed under the same license as the repo and I'm totally fine with this.

From what I read up, actually the copyright would even still remain with me without any information anywhere as long as I could proof that it comes from me (git for the win). But I'm not a lawyer and might be wrong in which case I'm still fine with it :nerd_face: